src/Tracing/Status.php

```diff
@@ -59,33 +59,35 @@ public function __construct(int $code, string $description = null)
         }
     }
 
-    public static function new(int $code, string $description = null) {
+    public static function new(int $code = self::OK, string $description = null) 
+    {
```

Comment:
nit - my linter likes us to have the braces on the newlines; I think we need to discuss this as a group.